splash/lua_modules/splash.lua

```diff
@@ -96,6 +113,14 @@ function Splash.create(py_splash)
       command = yields_result(command)
     end
 
+    if opts.can_raise_async then
+      command = raises_async(command)
+    end
+
+    if opts.multiple_return_values then
```

Comment:
if I'm not mistaken, opts.multiple_return_values and unpacks_multiple_return_values was deleted in master.